test/IRGen/jit_metadata_strategy_runtime.swift

```diff
@@ -0,0 +1,40 @@
+// RUN: %target-swift-frontend -interpret %s
+// REQUIRES: executable_test
+// REQUIRES: foundation
```

Comment:
You should use REQUIRES: OS=macosx. I don't think simulator or device testing supports the JIT.

Reply:
This one is `REQUIRES: swift_interpreter`.

Reply:
Ah, nice. I'll fix that up and cherry-pick this to the 5.1 branch.